### PR TITLE
PV is deprecated; no longer recommending it

### PR DESCRIPTION
**Pull request by inflatador**
PV is deprecated for security and performance reasons. Instead of recommending it, we replace this article with comparison to PVHVM and how to convert existing servers.

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-a8e38d3604/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-7f4ddf932b/how-to/).

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-71198d8830/how-to/)

File: content/cloud-servers/converting-your-server-from-pv-to-pvhvm.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+last_modified_date: '2018-03-05' 
+last_modified_by: Cat Lookabaugh 
+product: Cloud Servers 
+product_url: cloud-servers 
+---
+
+XenServer, which is the virtualization platform used by the Rackspace public cloud, supports multiple 
+virtualization modes. For better performance and security, convert your Rackspace cloud server from 
+the older paravirtualization (PV) mode to the newer paravirtualization with full hardware 
+virtualization (PVHVM) mode. Newer distributions, such as CentOS 7 and Ubuntu 16, are only offered
```
Comment (stephamon) [inline content/cloud-servers/converting-your-server-from-pv-to-pvhvm.md]: ... are offered as only PVHVM...

File: content/cloud-servers/converting-your-server-from-pv-to-pvhvm.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+- Debian 7
+
+If your server cannot be converted, we recommend that you deploy a new server from an existing base image and 
+migrate your data to the new server.
+
+To learn more about the conversion process, see [this Rackspace Community article](https://community.rackspace.com/general/f/general-discussion-forum/8315/rackspace-public-cloud-converting-pv-instance-to-pvhvm).
+
+Some advantages and disadavantages of PVHVM are described in the following sections.
+
+### Performance ###
```
Comment (stephamon) [inline content/cloud-servers/converting-your-server-from-pv-to-pvhvm.md]: All of these headers don't need the ending set of `###`

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-af4ceb6c51/how-to/)

**Comment by stephamon** [comment]
👍

**Pull request merged**